index.html

```diff
@@ -260,6 +260,33 @@ <h1>About Myself</h1>
         </div>
     </section>
 
+    <section id="contact-form">
+        <div class="column">
+            <div>
+                <h2>Contact Me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a
+                    project that needs coding.
+                    I’d love to help with it!</p>
+            </div>
+            <form action="https://formspree.io/f/xyyaojgk" method="post">
+                <ul>
+                    <li>
+                        <input type="text" class="form-input" name="name" id="name" placeholder="Lorem Ipsum" maxlength="30" required />
+                    </li>
+                    <li>
+                        <input type="email" class="form-input" name="email" id="email" placeholder="loremipsum@gmail.com" required />
+                    </li>
+                    <li>
+                        <textarea placeholder="Write your message here..." maxlength="500" name="message" rows="10" cols="30" required></textarea>
+                    </li>
+                    <li>
+                        <input type="submit" class="contact-btn" value="Get In Touch" />
+                    </li>
+                </ul>
+            </form>
```

Comment:
Great job on your project! I can see that your form is functioning perfectly. I just have two small suggestions to enhance the visual appeal of your project and make it look closer to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=87xOlduU06OkdGD1-0):


- [ ] Could you consider **decreasing the space between the paragraph and the form**? This will make the content more compact and improve the overall look and feel. 🎯
- [ ] Additionally, could you kindly **decrease the space between the form and your submit button**? This will create a more cohesive and visually appealing layout. 💯

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/235809620-7b181eeb-1f5e-4894-9849-63ffe318e445.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/235809770-99f7f3ff-f31d-42fa-aff4-6335d5048252.png)

  _Just a friendly reminder that adhering as closely as possible to the Figma template is a requirement for this project. While you've done a great job so far, it's important to keep the template in mind as you continue with your design._ 
  ![image](https://user-images.githubusercontent.com/107952724/235323387-c50f856e-6774-45ca-ae81-7304c8b62709.png)